### PR TITLE
[Validator] Missing translations for Greek (el)

### LINKED ISSUE #43719: Missing translations for Greek (el)
Hello,

There are some missing translation for Greek (el). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.el.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.el.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by agaktr**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #43719 
| License       | MIT

Greek Translation Addition.

### CODE REVIEW

**Review by derrabus — CHANGES_REQUESTED** [review]
Thank you for your contributon. Please remove all extra files from this PR and restore the newline@EOF of the updated language file.

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,4 @@ phpunit.xml
 composer.phar
 package.tar
 /packages.json
-/.phpunit
+/.phpunit
```
Comment (derrabus) [inline .gitignore]: Should be reverted.

**Comment by agaktr** [comment]
Not sure why PHPUnit tests fail, some issue with MongoDB or something?! Have not touched anything :_)

**Comment by derrabus** [comment]
> Not sure why PHPUnit tests fail, some issue with MongoDB or something?! Have not touched anything :_)

You can ignore test failures that are unrelated to you changes.

**Comment by agaktr** [comment]
Inform me if i have to do anything else :)

**Comment by derrabus** [comment]
Still the same. Please revert your changes to the `.gitignore` file and remove your `.idea` folder from the PR, please. We cannot merged your PR with all that noise in there.

**Comment by agaktr** [comment]
Terribly sorry... Not sure how they reappeared! I believe i fixed this now!

**Comment by derrabus** [comment]
There's still a change in the `.gitignore` file (missing new line at EOF). Revert that file too, please. The language file must be the only file changed by this PR.

**Comment by agaktr** [comment]
added it!

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @agaktr.

**Comment by agaktr** [comment]
Anytime! Actually now that I learned how to do the translations I could do that on any new GR content!

**Pull request merged**